index.html

```diff
@@ -4,20 +4,37 @@
 	<meta charset="UTF-8">
 	<link rel="stylesheet" href="style.css">
 	<link href="https://fonts.googleapis.com/css?family=Denk+One" rel="stylesheet">
-	<title>find Pokemon meetups!</title>
+	<title>Catch a Pokémon meetup</title>
 </head>
+
 <body>
-	<h1>Find Pokemon meetups near you!</h1>
+	<header>
+		<img class="logo" src="pictures/logo.png" alt="logo" width="250px">
+		<h1>Catch a Pokémon meetup</h1>
+		<img class="ball" src="pictures/pokeball-logo.png" alt="ball" width="100px">
```

Comment:
Inline styling can become very messy, and declaring the width within your style.css file can help with separation of concerns.